py/htm/advanced/regions/RawSensor.py

```diff
@@ -19,8 +19,8 @@
 
 from collections import deque
 from htm.bindings.regions.PyRegion import PyRegion
-from htm.bindings.engine_internal import Array
-import numpy as np
+
+from .import extractList
```

Comment:
nit, probably ok, but next time please use the absolute paths for imports. The code is then easier to read and move around.

Reply:
The reference emphasizes the close relationship of the method with the region implementations.  Its there to support them. Besides this is good Pythonic 3 usage. :-)